text/0000-clarify-api-namespace.md

```diff
@@ -0,0 +1,41 @@
+- Start Date: 2015-08-10
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+Clarify API description namespace and iron out issues which rose during API Blueprint serialization to API description namespace.
+
+# Motivation
+The purpose of this document is to clear inconsistencies in the format and make it easier to build tools around it.
+
+# Detailed design
+
+## Data Structure
+Since `Data Structure` inheriting from `Data Structure Element` of DataStructure Correct the `Data Structure` to the following:
+
+```apib
+# Data Structure (Element)
++ element: dataStructure (string, fixed)
++ content (Data Structure Element)
+```
+
+## Transition Example
+Value of `content` should be changed to `[]` instead of `null`.
+
+## Category
+Remove the "reserved" word from the description of `scenario` since it can be used as category semantics.
+
+## Copy
+We need to clarify what the relation between copy elements in `content` and `description` in `attributes` is. Unless specified otherwise, a copy element's content should represent the description of it's parent element and SHOULD be used instead of parent element's description metadata. `Copy Element` should be added to `content` of the following elements in API namespace:
```

Comment:
"should represent" :arrow_right: "represents" 
"API namespace" :arrow_right: "API Description namespace"  for consistency...